### PR TITLE
Add health check handler

### LINKED ISSUE #1039: /health endpoint
Add `/health` endpoint, returns:

- blockchain metadata
- size of unconfirmed txn pool
- version data
- number of open connections
- uptime

### PR DESCRIPTION
**Pull request by stgleb**
closes #1039

### CODE REVIEW

File: src/gui/health.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		metadata := gateway.GetBlockchainMetadata()
+
+		resp := HealthResponse{
+			BlockChainMetadata:  metadata,
+			VersionData:         Version,
+			UnconfirmedTxCount:  txnCount,
+			OpenConnectionCount: connectionCount,
+		}
+
+		wh.SendOr404(w, &resp)
```
Comment (therealssj) [inline src/gui/health.go]: Please see: https://github.com/skycoin/skycoin/issues/1021

**Comment by therealssj** [comment]
need to do `make format`

File: src/gui/health.go (hunk 1 above)
Comment (stgleb) [reply src/gui/health.go]: Thank you

File: src/gui/health.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package gui
+
+import (
+	"net/http"
+
+	wh "github.com/skycoin/skycoin/src/util/http"
+	"github.com/skycoin/skycoin/src/visor"
+)
+
+const Version = "0.21.1"
```
[the remark this answers is not in the document]
Comment (stgleb) [reply src/gui/health.go]: Thank you

**Comment by olemis** [comment]
refs #1039

File: src/gui/health.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+type HealthResponse struct {
+	BlockChainMetadata  *visor.BlockchainMetadata `json:"blockchain_metadata"`
+	VersionData         string                    `json:"version_data"`
```
[the remark this answers is not in the document]
Comment (stgleb) [reply src/gui/health.go]: Done

**Comment by gz-c** [comment]
Needs documentation in gui/README.md

File: src/gui/README.md [hunk 4]
```diff
@@ -1738,3 +1740,18 @@ result:
...
+```
+URI: /health
+Method: GET
+```
+
+example:
+
+```bash
+curl 'http://127.0.0.1:6420/health'
+```
```
Comment (gz-c) [inline src/gui/README.md]: Needs an example response like all of the other endpoints documented here

**Comment by gz-c** [comment]
#1200

**Pull request closed without merging**